tests/main/postrm-purge/task.yaml

```diff
@@ -4,19 +4,28 @@ systems: [-ubuntu-core-16-*]
 
 execute: |
     echo "When some snaps are installed"
-    . $TESTSLIB/snaps.sh
+    . "$TESTSLIB/snaps.sh"
```

Comment:
Mainly because it's more noise, increasing both the cost of reading and the cost of reviewing as we have dozens of lines in this PR which don't really change any logic.